File: oss-center/src/main/java/com/jmsoftware/maf/osscenter/read/entity/SerializableGenericResponse.java
package com.jmsoftware.maf.osscenter.read.entity;

import lombok.AccessLevel;
import lombok.Data;
import lombok.Setter;

import java.util.List;
import java.util.Map;

/**
 * Description: SerializableGenericResponse, change description here.
 *
 * @author Johnny Miller (锺俊), email: johnnysviva@outlook.com, date: 8/12/2021 5:07 PM
 **/
@Data
@Setter(AccessLevel.PROTECTED)
public class SerializableGenericResponse {
    private Map<String, List<String>> headers;
    private String bucket;
    private String region;
    private String object;
}

File: oss-center/oss-center-infra/src/main/kotlin/com/jmsoftware/maf/osscenter/response/SerializableGenericResponse.kt
package com.jmsoftware.maf.osscenter.response

/**
 * # SerializableGenericResponse
 * 
 * Description: SerializableGenericResponse, change description here.
 *
 * @author Johnny Miller (锺俊), e-mail: johnnysviva@outlook.com, date: 4/17/22 8:38 AM
 */
open class SerializableGenericResponse {
    var headers: Map<String, List<String>>? = null
    var bucket: String? = null
    var region: String? = null
    var `object`: String? = null
}
